release/release_tests.yaml

```diff
@@ -3986,6 +3986,27 @@
     type: sdk_command
     file_manager: sdk
 
+- name: many_nodes_multi_master_test
+  group: core-daily-test
+  working_dir: nightly_tests
+  legacy:
+    test_name: many_nodes_multi_master_test
+    test_suite: nightly_tests
+
+  frequency: nightly-3x
+  team: core
+  cluster:
+    cluster_env: many_nodes_tests/app_config.yaml
+    cluster_compute: many_nodes_tests/compute_config.yaml
+
+  run:
+    timeout: 7200
+    script: python many_nodes_tests/multi_master_test.py
+    wait_for_nodes:
+      num_nodes: 251
```

Comment:
can we test this in smaller # of nodes?

Reply:
This config is actually the same as the one of `actor_test.py`. They should have the same scale to make an apple-to-apple comparison.